aws-ebs-csi-driver.yaml

```
@@ -32,15 +32,11 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: e6dae6132fc8383340024440fc639a135bebd4ba
 
-  - runs: |
-      # Our global LDFLAGS conflict with a Makefile parameter
-      unset LDFLAGS
-
-      make ARCH=$(go env GOARCH)
-      mkdir -p ${{targets.destdir}}/usr/bin
-      cp bin/aws-ebs-csi-driver ${{targets.destdir}}/usr/bin/
-
-  - uses: strip
+  - uses: go/build
+    with:
+      packages: ./cmd
+      ldflags: -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.driverVersion=${{package.version}} -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud.driverVersion=${{package.version}} -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.gitCommit=$(git rev-parse HEAD) -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.buildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
```

nit: requesting to make this inline so that it's easier to edit. 
```suggestion
      ldflags: |
        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.driverVersion=${{package.version}} 
        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud.driverVersion=${{package.version}} 
        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.gitCommit=$(git rev-parse HEAD) 
        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.buildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")

```